setup.py

```diff
@@ -15,12 +22,14 @@
     url = 'https://bitbucket.org/kmike/yandex-maps/',
 
     description = 'Yandex.Maps API python wrapper with optional django integration.',
-    long_description = open('README.rst').read().decode('utf8') + open('CHANGES.rst').read().decode('utf8'),
+    long_description = open('README.rst').read() + open('CHANGES.rst').read(),
```

Comment:
It has nothing to do with setup.py - open("README.rst").read() will decode README.rst conntents using `locale.getpreferredencoding()`; it can fail with an exception on Windows or other OSes where locale is not UTF-8.

But setup.py is also tricky; it cares about long_description type and there are numerous bugs with non-ascii text. As I recall, in Python 2 it is required to be str (aka bytes), and in Python 3 it must be unicode. It may have changed now, but pip bugs are still open, and other projects like [requests](https://github.com/kennethreitz/requests/pull/2420) use workarounds. 

sys.setdefaultencoding was another workaround which seemed to work most of times in Python 2; I gave up trying to find a proper solution :)